intro-dnac/03_Path_Trace/path_trace.py

```diff
@@ -1,8 +1,19 @@
 import requests
 import json
 from requests.auth import HTTPBasicAuth
+import os
```

Comment:
Hey @klevenstein - I think more changes than just requirements.txt go in, is that intentional? Just want to check.

Reply:
@kiskander any comments?